tests/conftest.py

```diff
@@ -47,6 +50,15 @@ def db_migration():
     print("Done db setup")
 
 
+# Raise exceptions on _all_ requests made via the `requests` library - prevents accidentally calling external services
```

Comment:
` prevents accidentally calling external services`... unless you use curl :trollface: